### PR TITLE
added 'new' and 'version' feature

### PR DESCRIPTION
**Pull request by Adi2612**
@shubhodeep9 
I have added 'new' and 'version' commands in it . As mentioned in issue #13 . 
for 'new' I have just did git clone ... but I didn't checked whether git is installed or not on that operating system. I'll search more about it and can you please describe me what you mean by adding a "new controller" .

### CODE REVIEW

**Comment by shubhodeep9** [comment]
Hi @Adi2612 first of all thanks for the PR.
Let's start by git issue.
the thing I have in mind is, instead of cloning the repo, let's download the repo zip in /temp or equivalent they have in Windows. and unzip in the current location with the given app name.
So for example:
`torn new sampleApp` 
should download repo zip to /temp and then unzip it in working directory.

**Comment by shubhodeep9** [comment]
Secondly, `torn controller User` should create a UserController with sample template code in `Controller` directory.

**Comment by Adi2612** [comment]
> Hi @Adi2612 first of all thanks for the PR.
> Let's start by git issue.
> the thing I have in mind is, instead of cloning the repo, let's download the repo zip in /temp or equivalent they have in Windows. and unzip in the current location with the given app name.
> So for example:
> `torn new sampleApp`
> should download repo zip to /temp and then unzip it in working directory.

@shubhodeep9 
Yup, It is a great idea and thus it will also solves its dependency on git. 
Can U tell me more why my commits have failed all the tests / checks.

**Comment by shubhodeep9** [comment]
@Adi2612 for now ignore the tests.
The travis has been failing for a while.

**Comment by Adi2612** [comment]
@shubhodeep9 , I have made changes in for "new" command . Please review it.

File: torn/plugins/console.py [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
+import requests
+import zipfile
+__version__ = "0.0.4"
+
 def handler(args):
     if args.action[0] == 'run':
         runHandler()
+    elif args.action[0] == 'new':
+        #  for new app , cloning form main repository
+        print('creating new app...')
```
Comment (shubhodeep9) [inline torn/plugins/console.py]: Please make a separate function, like runHandler on L#9, for this section, for example: newHandler and then call it here

**Review by shubhodeep9 — CHANGES_REQUESTED** [review]

File: torn/plugins/console.py (hunk 1 above)
Comment (Adi2612) [reply torn/plugins/console.py]: okay , will do in that way. thanks btw

**Comment by Adi2612** [comment]
hey @shubhodeep9  created function for them.

**Review by shubhodeep9 — APPROVED** [review]
Cool 👍

**Comment by shubhodeep9** [comment]
@Adi2612 will you be working on controller too?

**Comment by Adi2612** [comment]
> @Adi2612 will you be working on controller too?

yup , I would love to work on that but yet I don't understand what U mean by creating a new controller . Can you please describe it more and also explain what will be its uses.

**Comment by shubhodeep9** [comment]
> > @Adi2612 will you be working on controller too?
> 
> yup , I would love to work on that but yet I don't understand what U mean by creating a new controller . Can you please describe it more and also explain what will be its uses.

Let's have the discussion on #13 
meanwhile I'll merge this PR

**Pull request merged**